docker/amazonlinux2.Dockerfile

```diff
@@ -4,8 +4,8 @@ RUN amazon-linux-extras enable python3.8
 
 RUN yum update -y && yum install -y git make python38 gcc10 gcc10-c++ binutils tar
 
-ARG node_version=22.15.0
-ARG node_sha256=4f2515e143ffd73f069916ecc5daf503e7a05166c0ae4f1c1f8afdc8ab2f8a82
+ARG node_version=20.19.1
```

Comment:
Didn't version 22 work?

Reply:
GitHub Actions runners start the Docker container mounting Node.js to `/__e/node20`. This directory is something we cannot change, thus by having Node.js 22 in our Docker container we would have to symlink `/__e/node20` to a directory containing a Node.js build. It just feels cleaner to me to use Node.js 20, especially since we don't have any special requirement, and GHA uses Node.js 20 anyway